@xen-orchestra/lite/src/components/vm/VmsActionsBar.vue

```diff
@@ -54,15 +57,37 @@
       </template>
     </MenuItem>
   </AppMenu>
+  <UiModal v-if="isDeleteModalOpen" :icon="faSatellite">
```

Comment:
This will allow to cancel the modal with escape key

```suggestion
  <UiModal
    v-if="isDeleteModalOpen"
    :icon="faSatellite"
    @close="closeDeleteModal"
  >
```